@xen-orchestra/web-core/lib/components/ui/label-value/UiLabelValue.vue

```diff
@@ -0,0 +1,146 @@
+<!-- v3 -->
+<template>
+  <div class="ui-label-value typo-body-regular">
+    <div class="ui-label-value-grid-container">
+      <div class="label">
+        {{ label }}
+      </div>
+      <div class="value">
+        <div class="valueAddonWraper">
+          <slot name="value">
+            <div
+              v-if="value && !Array.isArray(value)"
+              v-tooltip
+              :class="ellipsis == true ? 'text-ellipsis' : 'text-wrap'"
+            >
+              {{ value }}
+            </div>
+            <UiTagsList v-else-if="Array.isArray(value) && value.length > 0" class="tagList">
+              <UiTag v-for="tag in value" :key="tag" accent="info" variant="secondary">
+                {{ tag }}
+              </UiTag>
+            </UiTagsList>
+          </slot>
+          <div v-if="slots.addons">
+            <slot name="addons" />
+          </div>
+        </div>
+
+        <div
+          v-if="slots.actions || (copyValue != undefined && (Array.isArray(copyValue) ? copyValue.length > 0 : true))"
+          class="actions"
+        >
+          <VtsCopyButton
+            v-if="
+              copyValue !== undefined &&
+              copyValue !== '' &&
+              !slots.actions &&
+              (Array.isArray(copyValue) ? copyValue.length > 0 : true)
+            "
+            :value="Array.isArray(copyValue) ? copyValue.join(', ') : copyValue"
+          />
+          <slot name="actions" />
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import VtsCopyButton from '@core/components/copy-button/VtsCopyButton.vue'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import UiTag from '../tag/UiTag.vue'
+import UiTagsList from '../tag/UiTagsList.vue'
+
+const { value, copyValue } = defineProps<{
+  label: string
+  value?: string | string[]
+  ellipsis?: boolean
+  copyValue?: string | string[]
```

Comment:
```suggestion
  copyValue?: string | string[]
  autoLabel?: boolean
```

According to Clémence, Figma's CSS for `UiLabelValue` differs depending on whether you're in a side panel or the system tab, for example.

Therefore, I suggest adding a props to define the label size and avoid having a constant spacing between the label and its value.

The idea would be to call this props wherever you don't want to define a fixed size for the label.

**System tab :**

<img width="620" height="384" alt="image" src="https://github.com/user-attachments/assets/53a2cee6-7ad9-4318-966f-9bb9265dfb83" />

**SidePanel :** 

<img width="550" height="321" alt="image" src="https://github.com/user-attachments/assets/1c3be44e-88bf-47f1-bde4-6047990ccfc9" />